app/src/main/java/com/quran/labs/androidquran/presenter/translation/InlineTranslationPresenter.java

```diff
@@ -31,12 +31,12 @@
     this.quranSettings = quranSettings;
   }
 
-  public void refresh(VerseRange verseRange) {
+  public void refresh(List<String> orderedTranslationsFilesNames, VerseRange verseRange) { //  pass orderedTranslationsFilesNames here to return ordered Verses from DBs
     if (getDisposable() != null) {
       getDisposable().dispose();
     }
 
-    setDisposable(getVerses(false, getTranslations(quranSettings), verseRange)
+    setDisposable(getVerses(false, orderedTranslationsFilesNames, verseRange)
```

Comment:
maybe instead of this (we can revert this change back), we can update `getVerses`, since it's used by both the normal and inline translation views - let's update that `getVerses` to do the sorting instead at the end via:

```java
final LocalTranslation[] sortedTranslations = Arrays.copyOf(translations, translations.length);
Arrays.sort(sortedTranslations, new LocalTranslationDisplaySort());
```

just as the code in `TranslationView` is doing today - we can then remove that code from `TranslationView` and have it in one place.

Reply:
yes it's better

Reply:
if we add `isActive` column to translations table, it will be better , what do you think ?

Reply:
or we can reuse `activeTranslationsFilesNames` from
https://github.com/MohAbbasher/quran_android/blob/7819c5cf5e9f8f5299bf91b1eb45c229ffe5e29c/app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java#L1405

Reply:
adding a column makes sense especially now that the sorting info is in the database also - but in order to keep this patch small, reuse what's there for now and we can keep this for a new PR?